static/js/stream_data.js

```diff
@@ -410,13 +410,10 @@ exports.receives_notifications = function (stream_name, notification_name) {
     return page_params["enable_stream_" + notification_name];
 };
 
-const stream_notification_settings = [
-    "desktop_notifications",
-    "audible_notifications",
-    "push_notifications",
-    "email_notifications",
-    "wildcard_mentions_notify",
-];
+function get_stream_notification_settings() {
+    return settings_notifications.all_notifications.settings.stream_notification_settings.map(
+        (notification_name) => notification_name.replace("enable_stream_", ""));
+}
```

Comment:
Doing this extra operation doesn't seem worthy, you can just avoid it.

Reply:
Yeah, I agree.